src/js/components/Collapsible/__tests__/Collapsible-test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import 'jest-styled-components';
+import renderer from 'react-test-renderer';
+import { cleanup } from '@testing-library/react';
+
+import { Collapsible } from '..';
+import { Box } from '../../Box';
+import { Grommet } from '../../Grommet';
+
+describe('Collapsible', () => {
+  afterEach(cleanup);
+
+  test('direction', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Collapsible direction="horizontal">
+          <Box background="dark-1">Box Body</Box>
+        </Collapsible>
+        <Collapsible direction="vertical">
+          <Box background="dark-1">Box Body</Box>
+        </Collapsible>
+      </Grommet>,
```

Comment:
I'm thinking there is no point in testing `direction` prop if we are not testing it as part of the behavioral testing. currently running this code is not rendering anything, so the snapshot aren't valuable.
Here is a codesandbox example for what this code renders https://codesandbox.io/s/collapsible-imzuq?file=/index.js